### PR TITLE
More links in getting started guide

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/getting-started.html [hunk 1]
```diff
@@ -49,20 +49,48 @@
...
     the <a href="https://github.com/vespa-engine/sample-apps/tree/master/msmarco-ranking">MS Marco Ranking sample application</a>.
     </li>
   </ul>
 
+  <strong>Vector Search</strong>
+  <p>
+    Learn how to use Vespa Vector Search in the <a href="nearest-neighbor-search-guide.html">practical nearest neighbor search guide</a>.
+    It uses Vespa's support for <a href="nearest-neighbor-search.html">nearest neighbor search</a>, there is
+    also support for fast <a href="approximate-nn-hnsw.html">approximate nearest neighbor search</a> in Vespa. 
+    The guide covers combinig vector search with filters and how to perform hybrid search,
```
Comment (kkraune) [inline en/getting-started.html:64]:
```suggestion
    The guide covers combining vector search with filters and how to perform hybrid search, 
```

**Review by kkraune — APPROVED** [review]
Great!

**Comment by jobergum** [comment]
Too much? Open for discusssion - BUt I felt ML inference was missing, and also the great multi node templates, and the two latest practical guides.

**Comment by kkraune** [comment]
> Too much? Open for discusssion - BUt I felt ML inference was missing, and also the great multi node templates, and the two latest practical guides.

lets merge and see - can always trim it later

**Pull request merged**